src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -389,11 +427,315 @@ public void SarifTransformerTests_ToVersionOne_OneRunWithRules()
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""invocation"": {
+        ""commandLine"": ""CodeScanner @collections.rsp""
+      },
+      ""results"": [],
+      ""toolNotifications"": [
+        {
+          ""id"": ""CTN0001"",
+          ""message"": ""Run started."",
+          ""level"": ""note""
+        },
+        {
+          ""id"": ""CTN9999"",
+          ""ruleId"": ""C2152"",
+          ""physicalLocation"": {
+            ""uri"": ""file:///home/buildAgent/src/crypto/hash.cpp""
+          },
+          ""message"": ""Exception evaluating rule \""C2152\"". Rule disabled; run continues."",
+          ""level"": ""error"",
+          ""threadId"": 52,
+          ""time"": ""2016-07-16T14:18:43.119Z"",
+          ""exception"": {
+            ""kind"": ""ExecutionEngine.RuleFailureException"",
+            ""message"": ""Unhandled exception during rule evaluation."",
+            ""innerExceptions"": [
+              {
+                ""kind"": ""System.ArgumentException"",
+                ""message"": ""length is < 0""
+              }
+            ]
+          }
+        },
+        {
+          ""id"": ""CTN0002"",
+          ""message"": ""Run ended."",
+          ""level"": ""note""
+        }
+      ],
+      ""configurationNotifications"": [
+        {
+          ""id"": ""UnknownRule"",
+          ""ruleId"": ""ABC0001"",
+          ""message"": ""Could not disable rule \""ABC0001\"" because there is no rule with that id.""
+        }
+      ],
+      ""properties"": {
+        ""sarifv2/run"": ""{\""tool\"":{\""name\"":\""CodeScanner\"",\""semanticVersion\"":\""2.1.0\""},\""invocations\"":[{\""commandLine\"":\""CodeScanner @collections.rsp\"",\""toolNotifications\"":[{\""id\"":\""CTN0001\"",\""message\"":{\""text\"":\""Run started.\""},\""level\"":\""note\""},{\""id\"":\""CTN9999\"",\""ruleId\"":\""C2152\"",\""physicalLocation\"":{\""fileLocation\"":{\""uri\"":\""file:///home/buildAgent/src/crypto/hash.cpp\""}},\""message\"":{\""text\"":\""Exception evaluating rule \\\""C2152\\\"". Rule disabled; run continues.\""},\""level\"":\""error\"",\""threadId\"":52,\""time\"":\""2016-07-16T14:18:43.119Z\"",\""exception\"":{\""kind\"":\""ExecutionEngine.RuleFailureException\"",\""message\"":\""Unhandled exception during rule evaluation.\"",\""innerExceptions\"":[{\""kind\"":\""System.ArgumentException\"",\""message\"":\""length is < 0\""}]}},{\""id\"":\""CTN0002\"",\""message\"":{\""text\"":\""Run ended.\""},\""level\"":\""note\""}],\""configurationNotifications\"":[{\""id\"":\""UnknownRule\"",\""ruleId\"":\""ABC0001\"",\""message\"":{\""text\"":\""Could not disable rule \\\""ABC0001\\\"" because there is no rule with that id.\""}}]}],\""results\"":[]}""
       }
     }
   ]
 }";
+
+            VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_OneRunWithNotificationsButNoInvocations()
```

Comment:
It's not a great test name, because it _does_ contain an `invocation` object. But IMO we don't need this test. The previous test covered the ground.

Reply:
OK